src/main/scala/fr/polytechnique/cmap/cnam/etl/extractors/acts/DcirBiologyActExtractor.scala

```diff
@@ -0,0 +1,63 @@
+package fr.polytechnique.cmap.cnam.etl.extractors.acts
+
+import scala.util.Try
+
+import fr.polytechnique.cmap.cnam.etl.events.{BiologyDcirAct, DcirAct, EventBuilder, MedicalAct}
+import fr.polytechnique.cmap.cnam.etl.extractors.dcir.DcirExtractor
+import fr.polytechnique.cmap.cnam.etl.sources.Sources
+import org.apache.spark.sql.{DataFrame, Row}
+
+
+object DcirBiologyActExtractor extends DcirExtractor[MedicalAct] {
```

Comment:
This class has a lot of duplicate code for the MedicalActExtractor. Can't we merge them to a super class to avoid that duplication?

Reply:
Yes you are right. I changed the DcirMedicalActExtractor class into a DcirActExtractor trait and implemented two objects DcirMedicalActExtractor and DcirBiologyActExtractor. 
This seems to work welll.